addons/point_of_sale/models/res_config_settings.py

```diff
@@ -35,6 +35,7 @@ def _default_pos_config(self):
     module_pos_adyen = fields.Boolean(string="Adyen Payment Terminal", help="The transactions are processed by Adyen. Set your Adyen credentials on the related payment method.")
     module_pos_stripe = fields.Boolean(string="Stripe Payment Terminal", help="The transactions are processed by Stripe. Set your Stripe credentials on the related payment method.")
     module_pos_six = fields.Boolean(string="Six Payment Terminal", help="The transactions are processed by Six. Set the IP address of the terminal on the related payment method.")
+    module_pos_paytm = fields.Boolean(string="PayTM Payment Terminal", help="The transactions are processed by PayTM. Set your PayTM credentials on the related payment method.")
```

Comment:
Though this is important, let's not introduce a new field in stable. We'll introduce this new field in master during the forward port.

In order to use this new addon, the user has to manually install it (from the apps), and not via the pos settings.